src/Calculator/Controls/EquationTextBox.cpp

```diff
@@ -114,7 +114,7 @@ void EquationTextBox::OnKeyDown(KeyRoutedEventArgs ^ e)
 
 void EquationTextBox::OnLostFocus(RoutedEventArgs ^ e)
 {
-    if (!m_richEditBox->ContextFlyout->IsOpen)
+    if (m_richEditBox != nullptr && !m_richEditBox->ContextFlyout->IsOpen)
```

Comment:
this check needs to be added to OnKeyDown as well. I was able to crash the app because the richeditbox is null

Reply:
This sounds like it was caused by a local merge conflict from master, I merged and pushed again, could you give it another shot?